src/main/jbake/content/hawkular-services/docs/quickstart-guide/index.adoc

```diff
@@ -55,20 +55,273 @@ endif::[]
+
+----
+curl -u myUsername:myPassword -X POST http://localhost:8080/hawkular/alerts/import/all -d @includes/alert-definition.json \
+   -H "Content-Type: application/json" -H "Hawkular-Tenant: myTenant"
+----
+
+== Step 4: Test the alert trigger
+
+From now on, if we enter metrics that satisfy the condition, an email would be send (assuming an email server is available and configured).
+Let's try to enter more metrics, similar to what we did earlier.
+
+.metrics_day_2.json
+----
+include::includes/metrics_day_2.json[]
+----
+
+And execute the command to store the metrics (and will evaluate against our alert trigger conditions):
+
+----
+curl -u myUsername:myPassword -X POST http://localhost:8080/hawkular/metrics/gauges/temperature/raw -d @metrics_day_2.json \
+   -H "Content-Type: application/json" -H "Hawkular-Tenant: myTenant"
+----
+
+In the logs of Hawkular Services, you should see 3 INFO messages showing that the conditions were met such as:
+
+----
+11:59:37,361 INFO  [org.hawkular.alerts.actions.api] (Thread-251 (ActiveMQ-client-global-threads-1118700939)) HAWKALERT240001: Plugin [email] has received an action message: [BusActionMessage[action=Action[eventId='temperature-trigger-1472551176767-dc41aaf3-bdd7-4a89-a950-44dc92f10c8b', ctime=1472551176769, event=Alert [alertId=temperature-trigger-1472551176767-dc41aaf3-bdd7-4a89-a950-44dc92f10c8b, status=OPEN, ackTime=0, ackBy=null, resolvedTime=0, resolvedBy=null, context={}], result='null']]]
+11:59:37,385 INFO  [org.hawkular.alerts.actions.api] (Thread-242 (ActiveMQ-client-global-threads-1118700939)) HAWKALERT240001: Plugin [email] has received an action message: [BusActionMessage[action=Action[eventId='temperature-trigger-1472551176770-300fda0d-2c82-46e3-9f09-f4e9ed4ffa3a', ctime=1472551176771, event=Alert [alertId=temperature-trigger-1472551176770-300fda0d-2c82-46e3-9f09-f4e9ed4ffa3a, status=OPEN, ackTime=0, ackBy=null, resolvedTime=0, resolvedBy=null, context={}], result='null']]]
+11:59:37,384 INFO  [org.hawkular.alerts.actions.api] (Thread-253 (ActiveMQ-client-global-threads-1118700939)) HAWKALERT240001: Plugin [email] has received an action message: [BusActionMessage[action=Action[eventId='temperature-trigger-1472551176759-90040813-1b7a-4bda-ac65-1c7f673d4daa', ctime=1472551176760, event=Alert [alertId=temperature-trigger-1472551176759-90040813-1b7a-4bda-ac65-1c7f673d4daa, status=OPEN, ackTime=0, ackBy=null, resolvedTime=0, resolvedBy=null, context={}], result='null']]]
+----
+
+== More
+Congratulations, you just scratched the surface of Hawkular Services!
+We played with basic metrics and alert capabilities, both have more advanced features and we didn't tackle the inventory service in this getting started guide, we would suggest to have a look at their respective APIs: link:/docs/rest/rest-alerts.html[Alerting REST API], link:/docs/rest/rest-metrics.html[Metrics REST API] and link:/docs/rest/rest-inventory.html[Inventory REST API]
```

Comment:
Perhaps a reference to the developers guide http://www.hawkular.org/community/docs/developer-guide/alerts.html could also help here rather than only pointing to the REST API.
I think both URLs should be useful for new users.